mkdocs/docs/HPC/conda.md

```diff
@@ -0,0 +1,36 @@
+# Conda
+
+We do not recommend the use of conda environments on the HPC clusters. 
```

Comment:
Add something:
We do not forbid the use of conda, and in some situations (trying a software, make a new workflow, teaching a course, etc.) are perfectly acceptible to use it.

Reply:
added in [8a342c7](https://github.com/hpcugent/vsc_user_docs/pull/649/commits/8a342c76561b2613623ca9c1a8617be590514b34)